src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -178,27 +182,27 @@ const formConfig = {
         name: {
           path: 'your-information/name',
           title: 'Your name',
-          depends: formData => get('certifierRole', formData) === 'other',
+          depends: formData => get('certifierRole', formData) !== 'applicant',
```

Comment:
Now, if the user selects `applicant`, then in the attestation block at the end of the form we'll ask for the first applicant's name. Otherwise, we'll always ask for the name stored in `certifierName`. This is a minor bug fix.